backend/game/deck/deck_handler.py

```diff
@@ -68,6 +68,7 @@ def buy_card_from_store(self, index, purchasing_player: Player):
             purchasing_player.add_card(card_to_buy)
             self.__card_store.remove(card_to_buy)
             self.__fill_card_store()
+        return card_to_buy
```

Comment:
Not sure if this is expected or not

Reply:
Something keith added, I didn't have a problem with it so I left it. Might be to attach to front end.